doc/reference/storage_btrfs.md

```diff
@@ -20,6 +23,10 @@ Snapshots can be read-write or read-only.
 The `btrfs` driver in LXD uses a subvolume per instance, image and snapshot.
 When creating a new object (for example, launching a new instance), it creates a Btrfs snapshot.
 
+Btrfs doesn't natively support storing block devices.
+Therefore, when using Btrfs for VMs, LXD creates a big file on disk to store the VM.
+This approach is not very efficient and might cause issues when creating snapshots.
```

Comment:
What sort of issues?

Reply:
@stgraber ? In the video you mention issues that can occur when taking snapshots of the big file, but not what sort of issues. ;)